yamusic/playlists.go

```diff
@@ -253,6 +279,26 @@ func (s *PlaylistsService) Get(
 	return playlist, resp, err
 }
 
+func (s *PlaylistsService) GetByUserIDAndKind(
+	ctx context.Context,
+	userID string,
```

Comment:
Why string?

Reply:
Sometimes we don't know the user id of playlists, we know only "nickname", e.g. https://music.yandex.ru/users/inurrick/playlists/1003, where `inurrick` is "nickname" and `1003` is kind. This "nickname" can be used as userID.